### DISCUSSION TITLE
Will we ever be able to use a 7GB+ model to train on?

### DISCUSSION THREAD

**Discussion by wktra** [TheLastBen/fast-stable-diffusion#344] [Q&A]
I'd like to use the 14GB Waifu Diffusion model to train on, but this colab warns me to use a 4Gb ckpt

**Comment by TheLastBen** [comment]
if you have colab pro, you can do it without a problem, with Colab free, the RAM is limited to 12GB, and 14GB models, even 7GB won't fit.

**Reply by wktra** [reply]
I have colab pro! how would I go about adding a 7Gb model without getting that error? 
OMG, much appreciation!!!!! @TheLastBen

**Reply by TheLastBen** [reply]
you shouldn't get any error with colab pro, are you getting an error ?

**Reply by wktra** [reply]
yes! I uploaded the waifu diffusion 14GB model and it said something like "incorrect ckpt or use a 4G ckpt"

**Reply by wktra** [reply]
I should say that I linked to it from my google drive

**Reply by wktra** [reply]
(also, I chose compatibility mode) @TheLastBen

**Comment by TheLastBen** [comment]
copy the whole error log

**Reply by wktra** [reply]
I'll run it once more soon and post it here! thank you so much!!!!!!! @TheLastBen

**Reply by wktra** [reply]
@TheLastBen ran it again and it worked. 
Um, apparently, I was not using the waifu diffusion model. I was using a "proprietary" model (7GB) which should not be named. 
And for those wanting to try it with that particular larger model, it doesn't work and throws up an error. 
(Though it wasn't my original intent, I'm the kind of girl who tries to make something work when it doesn't work. Even if I don't much care for the end result. So, I tried the 7GB N**.ckpt on compatibility mode and it still didn't work)
